func_sig_registry/templates/base.html

```diff
@@ -16,7 +16,8 @@
         </div><!-- /.navbar-header -->	
         <div>
 					<ul class="nav navbar-nav">
-						<li><a href="{% url 'signature-list' %}">Browse Signatures</a></li>
+						<li><a href="{% url 'signature-list' %}">Browse Function Signatures</a></li>
```

Comment:
To reduce the size of these menu items (which I'm guessing are getting crowded) you might remove the word `"Browse"` from the text.